### DISCUSSION TITLE
Do not delete remember it is a great helper

### DISCUSSION THREAD

**Discussion by CrazyBoy49z** [koot-labs/telegram-bot-dialogs#25] [Ideas]
Do not delete remember it is a great helper
https://github.com/koot-labs/telegram-bot-dialogs/blob/master/src/Dialog.php#L203
And forget
https://github.com/koot-labs/telegram-bot-dialogs/blob/master/src/Dialog.php#L212

**Comment by lptn** [comment]
Hey @CrazyBoy49z 
Thank you for your feedback!

```diff
-$this->remember('key', 'value');
+$this->memory->put('key', 'value');

-$this->forget(key');
+$this->memory->forget('key');
```

Why I decided to remove this syntax sugar: sometimes you need more methods to work with "memory", but we can't keep adding syntax sugar for them and more and more duplicate existing Laravel collections functionality.
